backend/src/layer/data/swka_parser/test_data/util.rs

```diff
@@ -0,0 +1,54 @@
+#![allow(clippy::unwrap_used)]
```

Comment:
mark as cfg test
dont mix code with test files; create extra folder `test` instead

Reply:
until.rs seems to be the wrong name as this file contains "constant" data structs for tests. It could be better naming that file const_testing_data.rs. Then the positioning in this folder would be acceptable (?)

Reply:
Several tests in the production files need this data for testing. As there are no mocks (data-layer) some constant data ll be provided for testing. Same as with these html files.

Reply:
then please put this file in a seperate `tests` folder as test data does not contain any rust files

Reply:
or just put it next to the other files